danesfield/core/management/commands/populate_dev_data.py

```diff
@@ -0,0 +1,38 @@
+from pathlib import Path
+
+from django.core.files import File
+from django.core.files.base import ContentFile
+import djclick as click
+from rgd.models.common import ChecksumFile
+
+from danesfield.core.models import Dataset, DatasetRun
+
+TEST_DATA_DIRECTORY = (
+    Path(__file__).parent.parent.parent.parent.parent.resolve()
+    / 'dev'
+    / 'test_data'
+    / 'TilesetWithDiscreteLOD'
+)
+
+test_filenames = ['tileset.json', 'dragon_high.b3dm', 'dragon_low.b3dm', 'dragon_medium.b3dm']
+
+
+@click.command()
+def populate_dev_data():
+    point_cloud_file = ChecksumFile.objects.create(
+        name='TilesetWithDiscreteLOD.txt',
+        file=ContentFile("i'm a point cloud (not really)").read(),
```

Comment:
It seems that with this method, the file isn't actually uploaded to minio, which will cause an error if ever someone tried to download it. Can we just place the dummy point cloud file in the `TilesetWithDiscreteLOD` folder, and read it in from there?